akvo/iati/imports/mappers/ICCO/descriptions.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Akvo Reporting is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from ..descriptions import Descriptions
+
+description_start = dict(
+    project_name='project name:',
+    subtitle='subtitle:',
+    project_plan_summary='project summary:',
+    background='background:',
+    current_status='baseline situation:',
+    project_plan='project plan:',
+    sustainability='sustainability:',
+)
+
+def _text_starts_with(text, start):
+    return text.lower().startswith(start)
+
+class Descriptions(Descriptions):
```

Comment:
Also, this logic is not completely up-to-date anymore:
- Title field should only be stored from the title tag (we'll increase that to 60 so that all ICCO's titles will fit)
- Subtitle field should be the description starting with 'project name:'
- If the project plan summary is too long, it should be stored in the background field